### ISSUE TITLE
Lyrics scroll down to bottom and bug the whole song (i believe) when the first and last line are the same

### ISSUE THREAD

**Issue by ImZen0x** [surfbryce/beautiful-lyrics#133]
[example](https://streamable.com/e/kz5oul)
im not exactly sure why it does it im just guessing thats why

**Comment by surfbryce** [comment]
@ImZen0x I'll have to check the parsed data for this but I've ran into an issue like this before.

Because of a parsing issue (talking about issues prior) some lyrics during the song might last a long longer or start earlier than they are supposed to. Since the scrolling centers in-between active lyrics it can seem like its jumping/bugging out.

I'm assuming in this case the last lyric has a parser bug which is causing it to start a lot earlier and last until the end of the song - and since the scroller tries to keep in the center of the active lyrics, and the lyrics at the top parsed correctly - its trying to center between the bottom lyric and the top lyrics.

Again, I am making an assumption here but I wanted to let you know before I looked into it.
